.github/workflows/add all issues to project

```diff
@@ -13,6 +13,6 @@ jobs:
       - uses: actions/add-to-project@RELEASE_VERSION
```

Comment:
One other thing. I would change the file name to replace spaces with underscores or dashes. Whatever is consistent with the rest of the project

Reply:
Done!****

Reply:
This file still has spaces in it. Is that on purpose?

Reply:
If you look below, I updated the file to not have spaces.